controllers/substitute.js

```diff
@@ -141,7 +141,7 @@ function copyBaseFiles(passon) {
     return new Promise((fulfill, reject) => {
         let substFiles = passon.metadata.substitution.substitutionFiles;
         // check if aray substifutionFiles is not null
```

Comment:
The comment does not fit the tests anymore... also, maybe `Array.isArray(..)` might make this simpler and also return false for `undefined`.